x-pack/plugin/core/src/main/resources/fleet-filedelivery-data-ilm-policy.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
Can probably delete these ilm policies since we know we need to use another method for this in serverless